src/navigation/navigation.cpp

```diff
@@ -37,18 +37,18 @@ Navigation::Navigation(Logger& log, unsigned int axis/*=0*/)
            curr_msmt_(0),
            imu_reliable_ {{true, true, true, true}},
            nOutlierImus_(0),
-           stripe_counter_(0, 0),
+           stripe_counter_(log_, data_, displ_unc_,
+            velocity_uncertainty_, kStripeDistance),
```

Comment:
```suggestion
           stripe_counter_(log_, data_, displ_unc_, velocity_uncertainty_, kStripeDistance),
```
The line length limit is 100 ;)

I don't know the constructor initialiser syntax that well, but this seems a bit like `velocity_uncertainty_` is used here **before** it is initialised below (line 48). Can you check if this might actually cause issues?

Reply:
It's just a reference to velocity uncertainty, and since it is initialised immediately afterwards it runs fine. It is bad practice though and I'll change it so it works.